### ISSUE TITLE
DateTime incorrect after Export and Import

### ISSUE THREAD

**Issue by Accuraty** [2sic/2sxc#1800]
[x] bug report 
...about
[x] edit experience / UI
[x] admin experience UI
[x] Razor templating
[x] Content Types or data management

**Current behavior**
DateTime is unexpectedly outputting 5 hours in the future (which is also the current time zone and daylight savings time offset). We are setting a meeting date/time field in our own custom Meeting content-type to (Thu) 9/12/2019 at 19:00 (7pm) and it is outputting as "12:00 AM on Fri, Sep 13, 2019" on the page. Details below.

**Expected behavior**
Since we entered 7pm 9/12, we should output (see) 7pm 9/12.
 
**Minimal reproduction of the problem with instructions**
After upgrading site A to 2sxc v9.43.02 LTS (latest) and with site B using 2sxc v10.02-Pre, we exported our custom Meeting Content-Type from site A using Administration, App tab, Export parts. Selected just the one (Meeting) Content-Type, it's one Template, and NONE of the data (Entities). We then imported this successfully in to Site B. However, the Template did not come over (so that is a separate or related issue?), but the Meeting Content-Type did and appears to be a perfect match from Site A; all fields.
 
**Both sites** have DNN Site Settings/Site Time Zone set to (UTC-06:00) Central Time (US & Canada).

In Site B (2sxc 10.02-Pre), when we add or edit an item setting the date as follows:
![image](https://user-images.githubusercontent.com/653301/63801318-9ed72580-c8d5-11e9-9d9a-8e441e186651.png)

and then output it in the View using the following code:
```
                            <span @Edit.TagToolbar(meeting, actions: "edit", settings: new { align = "left", hover = "none" } )>
                                <strong>@meeting.MeetingCategory</strong>,
                                @meeting.MeetingDate.ToString("h:mm tt on ddd, MMM d" + ( showYear ? ", yyyy" : "" ))
                            </span>
```

we unexpectedly get:
![image](https://user-images.githubusercontent.com/653301/63801526-21f87b80-c8d6-11e9-92f7-3761febae032.png)

Please note that the exact same code and scenario on Site A where we imported the Content-Type from, stores and display the time as expected:
![image](https://user-images.githubusercontent.com/653301/63801663-6d128e80-c8d6-11e9-93ad-c69535e4b258.png)
 
**Please tell us about your environment:**
DNN 9.03.02
2sxc 10.00.02 Pre
Browser: all (n/a)
Language: any/all
 
So this is a pre-existing, working Content-Type/View on Site A; in production and live. Site B is an in progress, soon to go live project. I cannot find any difference in settings to explain why the output on  Site B is not automatically adjusting the time (and date) as normally expected.

**Comment by iJungleboy** [comment]
Thanks, this sounds like a time zone issue. We already fixed something in 2sxc 10 for this, so I hope it's automatically fixed in the next release next week. I'll keep it open till then.

**Comment by iJungleboy** [comment]
@PagnoDunadan should this be fixed with what we changed in the 2sxc 10 date/time picker?

**Comment by PagnoDunadan** [comment]
Versions 9 and 10 didn't handle datepicker in exactly the same way, but only for datepicker which didn't adjust hours as far as I'm aware.

Bug was reported and decision was to save date in UTC and ignore time zone offsets. This will be included in the next update but shouldn't affect this issue.

In this case, I don't know why razor seems to adjust timezone in version 9 and not in version 10, or if dates are indeed different.

You can check the exact saved value in browser developer tools.

**Version 9:**
![2019-08-29 10_00_33-My Website _ testing edit ui](https://user-images.githubusercontent.com/7113033/63922113-58540900-ca44-11e9-949f-21b62d6e4d6a.png)

**Version 10:**
![2019-08-29 10_01_09-Content Editing UI (Angular)](https://user-images.githubusercontent.com/7113033/63922129-5e49ea00-ca44-11e9-91e7-43747feda7d5.png)

I will double check and see if I can stumble upon the same issue you are having.

**Comment by Accuraty** [comment]
Ok, that was helpful, thank you. So now I can confirm in v10.02-Pre that when I enter and save any date/time it then stores it UTC. So the View is displaying it correctly, the v10 UI is storing it wrong. I think... 

So I edit an Entity, Set the Date/Time to 9/12/2019 7pm (19:00) and we are -5 UTC. I save, then I repeat your steps above and see this (image below), so it does appear that on save we are not time zone adjusting, so I think the View is doing its job correctly, the v10 UI is not.

![image](https://user-images.githubusercontent.com/653301/63945991-f8586500-ca39-11e9-8730-e2fab07e8d2a.png)

**Comment by PagnoDunadan** [comment]
You were close and helped me pinpoint the problem.

Version 9:
Save 2019-08-29 17:30 and backend will save 2019-08-29 17:30.
Template renders 2019-08-29 17:30
Open form again and date is 2019-08-29 17:30
Change to 2019-08-29 17:31 and backend will save 2019-08-29 17:31
All is fine here as there are no timezone calculations

Version 10
Save 2019-08-29 17:30 and backend will save 2019-08-29 15:30 (my timezone is GMT+2)
Template renders 2019-08-29 15:30
Open form again and date is 2019-08-29 15:30, NOT 2019-08-29 17:30
Change to 2019-08-29 15:31 and backend will save  2019-08-29 13:31

v10 saved time adjusted by timezone difference, but failed to adjust timezone when reading data. Dunno how this happens but last week v10 was changed to work with UTC and won't change dates anymore. This change will come with next version, but for the time being you probably have to fall back to v9 UI:

![2019-08-29 16_53_49-My Website _ DateTime Test](https://user-images.githubusercontent.com/7113033/63951264-b9e49980-ca7d-11e9-92ed-81f8ee417ec2.png)

![2019-08-29 16_53_14-My Website _ DateTime Test](https://user-images.githubusercontent.com/7113033/63951270-bcdf8a00-ca7d-11e9-9ff7-6956639dd731.png)

**Comment by iJungleboy** [comment]
I believe it's fixed in 10.3 - pls verify :)
